health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/CampaignBoundary.js

```
@@ -0,0 +1,811 @@
+    }, [boundaryHierarchy])
+
+    // Prevent unnecessary useEffect calls
+    useEffect(() => {
+        if (!boundaryHierarchy || boundaryHierarchy.length === 0) return;
+
+        let bH = [];
+        let top = null;
+        let dic = {};
+
+        for (const ob of boundaryHierarchy) {
+            let bType = ob["boundaryType"];
```

Biome

[error] This let declares a variable that is only assigned once.

'bType' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)